src/puppetlabs/rbac_client/services/activity.clj

```diff
@@ -11,19 +11,18 @@
 (defn v1->v2
   ;; make any adjustments needed to make a (maybe) v1 payload into a v2 payload
   [event-bundle]
-  (if (contains? event-bundle :object)
-    (-> event-bundle
-        (dissoc :object)
-        (assoc :objects [(:object event-bundle)]))
+  (if (get-in event-bundle [:commit :object])
```

Comment:
Might be a good opportunity to use `if-let` to avoid digging the value out twice:  https://clojuredocs.org/clojure.core/if-let

Reply:
ah, yeah, I'll try that